### ISSUE TITLE
Tripit api parameter: include_objects and object id

### ISSUE THREAD

**Issue by xinlishang** [tripit/api#104]
I am using JAVA binding examples provided here http://tripit.github.io/api/doc/v1/index.html to test my account. The response said bad request "you must specify an id of the lodging object you wish to retrieve". The document doesn't machine how to get the object ID. How does my app know it the object ID? 

Another thing is include_objects  parameter. It said if it set to true then the response from the API will include all objects. My understanding is I don't need object ID anymore. But it still response with bad request asking for object ID. 

Basically, I tried both URL as below. Both of them failed with above error. 
https://api.tripit.com/v1/get/lodging
https://api.tripit.com/v1/get/lodging/include_objects/true

**Comment by tariqislam** [comment]
Hi,

If you want to get a specific travel object, e.g. lodging, you have to specify the id:

https://api.tripit.com/v1/get/lodging/id/123

However, if you don't know the id of the object, you can use the /v1/list call to get all travel objects:

https://api.tripit.com/v1/list/object (please see the documentation: http://tripit.github.io/api/doc/v1/index.html#method_list)

Thanks.

Tariq

**Comment by xinlishang** [comment]
Thanks. It works. I am almost there! There is one more questions here. I just published a question in the forum "About the oauth_callback in the API". 

Date: Tue, 21 Jan 2014 23:14:44 -0800
